nextflow.config

```
@@ -48,7 +48,7 @@ params {
 
 // Container slug. Stable releases should specify release tag!
 // Developmental code should specify :dev
-process.container = 'qbicpipelines/rnadeseq:dev'
+process.container = 'tair_container'
```

The container needs to be changed back to the dev container.
```suggestion
process.container = 'qbicpipelines/rnadeseq:dev'
```